feature-libs/tms/public_api.ts

```diff
@@ -0,0 +1,5 @@
+/*
+ * Public API Surface of tms
+ */
+
+export * from './core/index';
```

Comment:
I would expect the structure to look like this:
```
@spartacus/tms/core
@spartacus/tms/gtm
@spartacus/tms/adobelaunch
```
Any opinions?

Reply:
Foundation done.
We can discuss it further (ll, core, root, etc.)